core-services/quay-mirroring/openshift/Makefile

```diff
@@ -0,0 +1,11 @@
+# Regenerate the on cluster image streams from the authoritative mirror
```

Comment:
Why is this file specific to quay?  We may very well have others, suggest this should be "image-mirroring"

Reply:
Yeah, why not. I'll rename it.

Reply:
Addressed in a followup https://github.com/openshift/release/pull/4401 that needs to merge after this one, because I need to fixup config-updater config too, which needs to happen in a PR separate from changing the content of the tracked files.